### ISSUE TITLE
MultiSelect DAP violation

### ISSUE THREAD

**Issue by SoSubtle** [carbon-design-system/carbon#5317]
## DAP Violation with MultiSelect 
[](url)
The MultiSelect component at [here](https://www.carbondesignsystem.com/components/dropdown/code/) is giving a DAP violation accessibility error titled "An interactive element must have an accessible name". Could this be an issue with the component or DAP ?
## Environment

> Operating system
Mac OSX

> Browser
Chrome

## Steps to reproduce the issue

1. Go to https://www.carbondesignsystem.com/components/dropdown/code/
2. Run a DAP Scan on the Multi Select component.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code

![image](https://user-images.githubusercontent.com/33182377/74239117-1c99bc80-4ccf-11ea-9a2c-66ac7ce45328.png)

-

**Comment by tw15egan** [comment]
Hey, thanks for reporting! This should be resolved in the next minor release (`v10.10.0`) as it was just fixed in this PR https://github.com/carbon-design-system/carbon/pull/5039

**Related issues**
- https://github.com/carbon-design-system/carbon/issues/4942
- https://github.com/carbon-design-system/carbon/issues/4008

Going to close this as resolved, but let me know if you still encounter this issue once `10.10.0` is released and you've updated.

**Comment by teresamonahan** [comment]
@tw15egan Thanks for this info. Any idea when `10.10.0` will be released? Thanks
